src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php

```diff
@@ -28,6 +32,57 @@ class DoctrinePHPCRDriver extends AbstractDoctrineDriver
     /**
      * {@inheritdoc}
      */
+    public function load(ContainerBuilder $container, MetadataInterface $metadata)
+    {
+        parent::load($container, $metadata);
+        $this->addResourceListeners($container, $metadata);
+    }
+
+    /**
+     * Add resource event listeners.
+     */
+    protected function addResourceListeners(ContainerBuilder $container, MetadataInterface $metadata)
+    {
+        $createEventName = sprintf('%s.%s.pre_%s', $metadata->getApplicationName(), $metadata->getName(), 'create');
+        $updateEventName = sprintf('%s.%s.pre_%s', $metadata->getApplicationName(), $metadata->getName(), 'update');
+
+        // default path listener
+        $defaultPath = new Definition(DefaultPathListener::class);
+        $defaultPath->setArguments([
+            new Reference('sylius.resource_registry'),
+            new Reference($metadata->getServiceId('manager'))
+        ]);
+        $defaultPath->addTag('kernel.event_listener', [
+            'event' => $createEventName,
+            'method' => 'onPreCreate'
+        ]);
+        $container->setDefinition(
+            'sylius.resource.doctrine.odm.phpcr.event_listener.default_path',
```

Comment:
Looks like this is the wrong thing to do.. I guess there should be a Listener for every `$metadata` instance? This would also simplify things.